src/components/app/DocumentList.js

```diff
@@ -195,20 +195,22 @@ class DocumentList extends Component {
             viewId
         } = this.state;
 
-        dispatch(initLayout(
-            'documentView', windowType, null, null, null, null, type, true
-        )).then(response => {
-            this.setState({
-                layout: response.data
-            }, () => {
-                if(viewId && !isNewFilter){
-                    this.browseView();
-                }else{
-                    this.createView();
-                }
-                setModalTitle && setModalTitle(response.data.caption)
-            })
-        });
+        if(windowType){
```

Comment:
changed solution, the problem was because SideList was opened after changing document view, and windowType prop was undefined